frontend/src/app/modules/online-editor/online-editor-page/online-editor-page.component.ts

```diff
@@ -84,6 +101,9 @@ export class OnlineEditorPageComponent implements OnDestroy, OnInit {
 
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
         (<any>window).monaco.editor.setModelLanguage((<any>window).monaco.editor.getModels()[0], selectedLang);
+
+        this.languageVersions = this.getLanguageVersionsByLanguage(selectedLang);
+        [this.selectedLanguageVersion] = this.languageVersions;
```

Comment:
strange way to assign selected value

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/68563967/61618a81-ba0f-4326-b49b-66d70ca89094)

Reply:
this way was already used in dev-branch and i just copied